### PR TITLE
Player status GUI now functional

### PR DESCRIPTION
**Pull request by DazKins**

### CODE REVIEW

File: Scripts/Player.cs [hunk 1]
```diff
@@ -173,4 +219,19 @@ public override void _Process(float delta)
 
         onFloor = this.IsOnFloor();
     }
+
+    public float GetCurrentAir()
```
Comment (JamesHinshelwood) [inline Scripts/Player.cs]: Can we use properties for these to keep consistency with the rest of the code. e.g. public float CurrentAir { get; set; } or something

Comment (JamesHinshelwood) [inline Scripts/Player.cs]: Also, getting this error
image

**Review by JamesHinshelwood — CHANGES_REQUESTED** [review]

**Comment by DazKins** [comment]
I'll be honest, I don't see the point of using the properties much in C#, surely it just makes code within the class really messy because of the getters and setters, this feels like a much more natural way to do things. The error is due to some parts of the code, trying to remove a child from a node that is null, this is fine and will not cause the code to crash but I will add some null checks

**Comment by DazKins** [comment]
Just found another bug with the inventory screen, so don't merge plz

**Comment by JamesHinshelwood** [comment]
They make the interface of the class much cleaner to use. You don't need loads of GetVariable SetVariable functions, you can just use the variable

**Review by JamesHinshelwood — APPROVED** [review]
Nice work!

**Comment by JamesHinshelwood** [comment]
Closes #16

**Pull request merged**